SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -2464,13 +2478,16 @@ \subsection{Alignment}
 
 \small
 \begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
-InputPixelType  m_ForegroundValue;
-OutputPixelType m_BackgroundValue;
+bool                        m_Quiet{ false };
+bool                        m_TestAbort{ false };
+std::string                 m_Comment;
+itk::ProcessObject::Pointer m_Process;
 
-unsigned int m_MaximumIterations;
+CommandType::Pointer m_StartFilterCommand;
+CommandType::Pointer m_EndFilterCommand;
 
-std::vector<double> m_Sensitivity;
-std::vector<float>  m_Overlaps;
+unsigned long m_StartTag{ 0 };
```

Comment:
These 2 lines make the example longer, while not contributing significantly to the point. Maybe remove them?

Reply:
Resolving since I believe 55ea845 now demonstrates the point well.